mission_control/navigator_launch/launch/gnc.launch

```diff
@@ -17,7 +17,7 @@
 
 
     <node name="wrench_arbiter" ns="/wrench" pkg="topic_tools" type="mux"
-          args="cmd autonomous rc emergency keyboard" >
+          args="cmd cmd autonomous rc emergency keyboard" >
```

Comment:
why?

Reply:
first argument is output of mux, the rest of the arugments are inputs to the mux. The first input is used as the initial. So the system will start by forwarding /wrench/cmd to /wrench/cmd, so not accepting any of the actual wrenches.

Reply:
it could also be ```cmd dummy_fake_wrench autonomous rc emergency keyboard```

Reply:
Ah, okay